subsys/bluetooth/controller/Kconfig

```diff
@@ -824,8 +838,6 @@ config BT_RX_USER_PDU_LEN
 
 endmenu
 
-comment "BLE Controller hardware configuration"
```

Comment:
Add this "comment" (that acts as separators) back

Reply:
Done